networks/__init__.py

```diff
@@ -0,0 +1,12 @@
+import torch.nn as nn
+
+from networks.caml import CAML
+from networks.kim_cnn import KimCNN
+from networks.xml_cnn import XMLCNN
+
+
+def get_init_weight_func(config):
+    def init_weight(m):
+        if isinstance(m, nn.Linear) or isinstance(m, nn.Conv2d):
+            getattr(nn.init, config.init_weight+ '_')(m.weight)
```

Comment:
If  the "if" statement fails (e.g., Conv1d above), should we issue an error/warning message?

Reply:
For the rest of the modules like BatchNorm, Dropout, etc, people usually use the default initialization.
So I think we don't need to raise an error/warning here.

Reply:
OK